src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1000,6 +1005,22 @@ public void UpdateMicrobePartSelections()
         }
     }
 
+    /// <summary>
+    ///   Updates the visibility of the move cancel button.
+    /// </summary>
+    public void UpdateCancelButtonVisibility()
+    {
+        // Disables if we're not moving an organelle, otherwise we're moving one and the cancel button is active
```

Comment:
Deprecated comment.

Reply:
This comment has been removed in the updated code, it doesn't seem needed.